### PR TITLE
Enhancement/147 enable hive integration test

### PR DESCRIPTION
**Pull request by snehlsen**
Enabled Hive Integration Test (#147):
* enabled automated hive integration test using docker-compose (base container https://hub.docker.com/r/bde2020/hive/)
* added automated schema creation to hive IT
* added join tests to hive IT
* adapted existing hive integration tests to new schema
* changed default driver directory for easier local setup

**Limitation**
This test cannot run on TravisCI, since the used JDBC driver is proprietary (Cloudra/Simba Hive JDBC driver).

### CODE REVIEW

File: jdbc-adapter/integration-test-data/socker.py [hunk 1]
```diff
@@ -30,13 +33,33 @@ def docker_run(config):
...
+    run(cmd)
+    cmd = "echo -e '\nnetworks:\n  default:\n    external:\n      name: {network}\n' >> docker-hive/docker-compose.yml".format(
+        network = DOCKER_NETWORK
+    )
+    print(cmd)
+    run(cmd)
+    cmd = "sudo docker-compose up -d"
+    print(cmd)
+    run(cmd, "./docker-hive")
+    time.sleep(90)
```
Comment (redcatbear) [inline jdbc-adapter/integration-test-data/socker.py]: As you know using `sleep` to wait for a condition is always the second best approach. Any chance to at least assert the condition you are waiting for after the sleep and bail out if it is not fulfilled?

File: jdbc-adapter/virtualschema-jdbc-adapter/pom.xml [hunk 2]
```diff
@@ -14,6 +14,17 @@
...
             <id>it</id>
             <dependencies>
+                <dependency>
+                    <scope>verify</scope>
+                    <groupId>org.postgresql</groupId>
+                    <artifactId>postgresql</artifactId>
+                    <version>42.2.5</version>
+                </dependency>
+                <dependency>
+                    <groupId>org.apache.hive</groupId>
```
Comment (redcatbear) [inline jdbc-adapter/virtualschema-jdbc-adapter/pom.xml]: Which scope is this?

Comment (snehlsen) [reply jdbc-adapter/virtualschema-jdbc-adapter/pom.xml]: This is the profile *IT* which is used in the integration test. Before this was a general dependency which makes no sense.

File: jdbc-adapter/integration-test-data/socker.py (hunk 1 above)
Comment (snehlsen) [reply jdbc-adapter/integration-test-data/socker.py]: You can use `docker-compose ps` or `docker ps` but that the containers are running does not mean that the Hive cluster is ready to accept connections. You would have to observe the log output of the (right) container to know when it is ready. I don know enough about hive to implement this.

**Pull request merged**